lib/world.rb

```diff
@@ -9,6 +9,17 @@ def as_json
     @_wjson ||= JSON.parse(File.read(file), symbolize_names: true)
   end
 
+  # super-simplistic adapter for the inner data. Over time it might make
+  # sense to properly extract this to its own class, but for now we just
+  # want to tidy the interface up a little so that this is substitutable
+  # for an `everypolitician-ruby` Country in a few well-defined places.
+
+  Country = Struct.new(:url, :name, :names)
```

Comment:
We have a little bit of a chicken-and-egg situation here in that the 'url' method should probably be 'slug', but we need to match the interface of the (adapted) countries.json to which we add a `url` as well. We _could_ clean [that](https://github.com/everypolitician/viewer-sinatra/blob/c748ec2bcfd011f998b7b85ff38ee56051c7b46b/app.rb#L24) up first, but then we'd need to change the templates too, and then we're out in the fields playing a gramophone at the bull.

This version seems like it will let us make https://github.com/everypolitician/viewer-sinatra/pull/14617 simpler, and then we can switch both versions to be `.slug` in one go at the end when everything is neatly harmonised.